testing/sdk_install.py

```diff
@@ -176,41 +175,29 @@ def uninstall(
         zk=None):
     start = time.time()
 
-    global _installed_service_names
-    try:
-        _installed_service_names.remove(service_name)
-    except KeyError:
-        pass  # allow tests to 'uninstall' up-front
-
     log.info('Uninstalling {}'.format(service_name))
 
     try:
         retried_uninstall_package_and_wait(package_name, service_name=service_name)
     except Exception:
-        log.info('Got exception when uninstalling {}'.format(service_name))
-        log.info(traceback.format_exc())
+        log.exception('Got exception when uninstalling {}'.format(service_name))
         raise
```

Comment:
Does this raise the last thrown exception? Would it make sense to change this to:
```
except Exception as e
    ... 
    raise e
```

Reply:
It rethrows the original exception without messing with the stack of the original: https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement, https://stackoverflow.com/questions/18001721/raise-with-no-argument